obfx_modules/gutenberg-blocks/build/style.css

```diff
@@ -1,12 +1,849 @@
+    display: block;
+    background-color: #fefffa;
+    margin: 0 !important;
+    padding: 0;
+    cursor: pointer
+}
+
+.wp-block-orbitfox-accordion-area ul li .accordion-content {
+    color: rgba(48, 69, 92, 0.8);
+    font-size: 17px;
+    line-height: 26px;
+    letter-spacing: 1px;
+    position: relative;
+    margin-top: 14px !important;
+    max-height: 800px;
+    opacity: 1;
+    -webkit-transform: translate(0, 0);
+    transform: translate(0, 0);
+    z-index: 2
+}
+
+.wp-block-orbitfox-accordion-area ul li i {
+    position: absolute;
+    -webkit-transform: translate(-6px, 0);
+    transform: translate(-6px, 0);
+    margin-top: 16px;
+    right: 0
+}
+
+.wp-block-orbitfox-accordion-area ul li i:before, .wp-block-orbitfox-accordion-area ul li i:after {
+    content: "";
+    position: absolute;
+    background-color: rgba(48, 69, 92, 0.8);
+    width: 3px;
+    height: 9px
+}
+
+.wp-block-orbitfox-accordion-area ul li i:before {
+    -webkit-transform: translate(-2px, 0) rotate(45deg);
+    transform: translate(-2px, 0) rotate(45deg)
+}
+
+.wp-block-orbitfox-accordion-area ul li i:after {
+    -webkit-transform: translate(2px, 0) rotate(-45deg);
+    transform: translate(2px, 0) rotate(-45deg)
+}
+
+.wp-block-orbitfox-accordion-area ul li input[type=checkbox] {
+    position: absolute;
+    cursor: pointer;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    opacity: 0
+}
+
+.wp-block-orbitfox-accordion-area ul li input[type=checkbox]:checked ~ .accordion-content {
+    margin: 0 !important;
+    max-height: 0;
+    opacity: 0;
+    -webkit-transform: translate(0, 50%);
+    transform: translate(0, 50%)
+}
+
+.wp-block-orbitfox-accordion-area ul li input[type=checkbox]:checked ~ i:before {
+    -webkit-transform: translate(2px, 0) rotate(45deg);
+    transform: translate(2px, 0) rotate(45deg)
+}
+
+.wp-block-orbitfox-accordion-area ul li input[type=checkbox]:checked ~ i:after {
+    -webkit-transform: translate(-2px, 0) rotate(-45deg);
+    transform: translate(-2px, 0) rotate(-45deg)
+}
+
+.wp-block-orbitfox-pricing-table {
+    display: flex;
+    position: relative
+}
+
+.wp-block-orbitfox-pricing-table.is-dim:before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    background-color: rgba(0, 0, 0, 0.5)
+}
+
+.wp-block-orbitfox-pricing-table.is-dim .wp-block-column {
+    z-index: 1
+}
+
+.wp-block-orbitfox-pricing-table.is-parallax {
+    background-attachment: fixed !important
+}
+
+.wp-block-orbitfox-pricing-table.alignfull {
+    padding: 100px
+}
+
+.wp-block-orbitfox-pricing-table.alignwide {
+    padding: 100px 150px
+}
+
+.wp-block-orbitfox-pricing-table .wp-block-column {
+    position: relative;
+    flex: 1;
+    padding: 20px 10px;
+    margin: 0 20px
+}
+
+.wp-block-orbitfox-pricing-table .raised {
+    box-shadow: 0 16px 38px -12px rgba(0, 0, 0, 0.56), 0 4px 25px 0 rgba(0, 0, 0, 0.12), 0 8px 10px -5px rgba(0, 0, 0, 0.2)
+}
+
+.wp-block-separator {
```

Comment:
@radu-themeisle 
40px margin for the `.wp-block-separator` is a bit too much. Also, it's not ok to style it directly as this is rendered as a simple separator. It should be specifically selected for the pricing table by prefixing it properly `.wp-block-orbitfox-pricing-table .wp-block-separator`

I'd also remove the margin-bottom for the paragraphs used as properties in the tables. 

![image](https://user-images.githubusercontent.com/15010186/47556005-facbc900-d915-11e8-8c90-0844f1459ad0.png)